content/news/translation/2023/do-you-speak-prestashop-september-2023.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "Do You Speak PrestaShop? – September 2023 Edition"
+subtitle: "Where you'll learn that new strings are ready for translation"
+date: 2023-10-04
+authors: [JulieVarisellaz]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it's also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the software translations evolved in September 2023.
+
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+## New strings are ready for translation
+
+As you may have noticed, new strings were added to Crowdin on September 28. It's usually not possible to add new strings outside the release of a minor or major version of PrestaShop but we had to do it to fix a [bug](https://github.com/PrestaShop/PrestaShop/issues/32212).
+To go back to a 100% completion rate, those strings will need to be translated and validated. You can find the untranslated strings in the 8 file. Your project needs you! :muscle: 
```

Comment:
```suggestion
To go back to a 100% completion rate, those strings will need to be translated and validated. You can find the untranslated strings in the 8 files. Your project needs you! :muscle: 
```

Reply:
Does this edit make sense @Julievrz ?

Reply:
Mhm, not really, there's only one file named "8". ☺️

Reply:
Ah! Perfect I understand better now, I've reverted to what you wrote and just added some quotes.